### PR TITLE
150 bug application window sizing

### PR DESCRIPTION
**Pull request by RagnarTheEverchosen**
temp fix to window sizing and scaling

### CODE REVIEW

**Comment by RagnarTheEverchosen** [comment]
There are still some unresponsive elements. Those are not a concern of this branch and will be fixed down the road.

File: coh2-battlegrounds/coh2-battlegrounds/Utilities/Converters/IsGreaterThanConverer.cs [hunk 1]
```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using System.Globalization;
+using System.Windows.Data;
+
+namespace BattlegroundsApp.Utilities.Converters;
+public class IsGreaterThanConverer : IValueConverter {
+
+    public static readonly IValueConverter Instance = new IsGreaterThanConverer();
```
Comment (JustCodiex) [inline coh2-battlegrounds/coh2-battlegrounds/Utilities/Converters/IsGreaterThanConverer.cs:8]: Why a singleton instance?

File: coh2-battlegrounds/coh2-battlegrounds/Utilities/Converters/IsGreaterThanConverer.cs [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+        double doubleValue = System.Convert.ToDouble(value);
+        double compareToValue = System.Convert.ToDouble(parameter);
+
+        return doubleValue > compareToValue;
+
+    }
+
+    public object ConvertBack(object value, Type targetType, object parameter, CultureInfo culture) {
+
+        throw new NotImplementedException();
```
Comment (JustCodiex) [inline coh2-battlegrounds/coh2-battlegrounds/Utilities/Converters/IsGreaterThanConverer.cs:21]: Not implemented or not supported?

File: coh2-battlegrounds/coh2-battlegrounds/Utilities/Converters/IsLessThanConverter.cs [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+
+    }
+
+    public object ConvertBack(object value, Type targetType, object parameter, CultureInfo culture) { 
+    
+        throw new NotImplementedException();
+
+    }
+
+}
```
Comment (JustCodiex) [inline coh2-battlegrounds/coh2-battlegrounds/Utilities/Converters/IsLessThanConverter.cs:26]: Same as with IsGreaterThanConverter (Where are these used?)

File: coh2-battlegrounds/coh2-battlegrounds-editor/Pages/CompanyBrowserView.xaml [hunk 4]
```diff
@@ -156,85 +155,105 @@
...
+
+            <Grid.ColumnDefinitions>
+                <ColumnDefinition Width="auto"/>
+                <ColumnDefinition Width="auto"/>
+                <ColumnDefinition Width="auto"/>
+                <ColumnDefinition Width="auto"/>
+                <ColumnDefinition Width="auto"/>
+                <ColumnDefinition Width="auto"/>
+                <ColumnDefinition Width="auto"/>
+                <ColumnDefinition Width="auto"/>
```
Comment (JustCodiex) [inline coh2-battlegrounds/coh2-battlegrounds-editor/Pages/CompanyBrowserView.xaml:179]: Is the amount of column definitions necessary?

**Review by JustCodiex — APPROVED** [review]

**Pull request merged**